.github/workflows/qiita-plugin-ci.yml

```diff
@@ -62,8 +62,9 @@ jobs:
 
           # Setting up main qiita conda environment
           conda config --add channels conda-forge
-          conda create -q --yes -n qiita python=3.6 pip libgfortran numpy nginx cython redis
+          conda create -q --yes -n qiita python=3.6 pip==9.0.3 libgfortran numpy nginx cython redis
```

Comment:
why pin pip?

Reply:
It the "simplest" solution I have found [to this error](https://github.com/qiita-spots/qp-qiime2/runs/4245456034?check_suite_focus=true#step:5:1436); basically pin pip so conda can do it's thing and then update.

Reply:
It the "simplest" solution I have found [to this error](https://github.com/qiita-spots/qp-qiime2/runs/4245456034?check_suite_focus=true#step:5:1436); basically pin pip so conda can do its thing and then update.

Reply:
It the "simplest" solution I have found [to this error](https://github.com/qiita-spots/qp-qiime2/runs/4245456034?check_suite_focus=true#step:5:1436); basically pin pip so conda can do its thing and then update.

Reply:
that's frustrating, hope they fix soon